README.md

```diff
@@ -112,6 +112,7 @@ This is a curated list of tools, resources, and services supporting the Digital
 ## Other Resources
 
 - [Awesome Digital History](https://github.com/maehr/awesome-digital-history) - A curated list of awesome things related to digital history.
+- [Awesome IIIF](https://github.com/IIIF/awesome-iiif) - A curated list of awesome resources related to the International Image Interoperability Framework (IIIF)
```

Comment:
@julsraemy Add a dot at the end of the line to make the linter happy. Thanks for the contribution!

Reply:
Ok!